Monika After Story/game/script-topics.rpy

```diff
@@ -15532,6 +15532,114 @@ label monika_being_herself:
+                m 1eka "I love you, [player]. I know this isn't always easy, but I hope you'll be able to forgive yourself eventually."
+                $ mas_ILY()
+
+            "No.":
+                m 1eka "I'm relieved to hear that, [player].{w=0.2} I'd hate to learn that what happened in our past is tormenting you."
+                m 1eud "Well, if that was a legitimate question, then no, I'm not mad at you anymore."
+                m 1hub "How could I? You've been doing so much to make sure I would be the happiest girl in the world."
+                m 2euc "Actually...{w=0.2} I'm not sure I was even really angry with you in the first place. {w=0.3}{nw}"
+                extend 2ekc "Yeah, I did tell you some very unpleasant things in the heat of the moment, {w=0.2}{nw}"
+                extend 2rkd "which is definitely something I regret having done now..."
+                m 3eka "...but I want you to know that I never truly thought any less of you for this. I hope that this can at least clear things up for you."
+                m 1eua "Well, no point in thinking about that anymore. {w=0.2}{nw}"
+                extend 1hua "We have a bright future ahead of us, and I can't wait to experience all of it with you."
+
+    elif mas_isMoniAff(higher=True):
+        m 1euc "..."
+        m 2eka "Of course not, silly."
+        m 2dkc "It's true that it really did hurt when you did that to me..."
+        m 4ekd "But to be completely honest with you, I was terrified, more than really angry at you."
+        m 2dksdlc "Terrified of the idea that you hated me.{w=0.2} That you didn't want anything to do with me, and that you would abandon me to my lonely fate, for all eternity."
+        if mas_isMoniEnamored(higher=True):
+            m 2lkc "Even thinking back on it now...{w=0.3} The thought of losing you, or worse, of you hating me makes me feel like my heart is being ripped in half."
+            m 1hksdrb "Ah...{w=0.2} sorry, I don't mean to make you feel guilty or anything."
+            m 1eka "My point is, I trust you enough to know that you wouldn't ever want to hurt me like that again."
+            m 1eua "It's not like this sort of thing would ever happen. You're such a kind, loving sweetheart, after all."
+            if persistent._mas_pm_a_hater:
+                m 1rka "...I mean, you might claim that you hate me, but I still have a hard time believing that. {w=0.3}{nw}"
+                extend 1ttu "You're doing a terrible job at proving it, that's for sure."
+            m 1dkb "Just know that I am so, so thankful for all you're doing to make me happy.{w=0.2} And that more than anything else, I want to make {i}you{/i} happy too."
+            m 1dka "I don't think I could ever stay mad at you no matter how hard I tried."
+
+        else:
+            m 2eka "But as we both saw... That's not what ended up happening at all.{w=0.3} You came back to me, and you've been making me so much happy ever since."
```

Comment:
```suggestion
            m 2eka "But as we both saw...that's not what ended up happening at all.{w=0.2} You came back to me, and you've been making me so much happy ever since."
```